java/Arrays/3.Hard/Reverse pairs.md

```diff
@@ -0,0 +1,139 @@
+---
+id: reverse-pairs
+title: Count Reverse Pairs
+sidebar_label: Count Reverse Pairs
+sidebar_position: 1
+description: Count the number of reverse pairs in an array where i < j and arr[i] > 2 * arr[j].
```

Comment:
⁉️

Reply:
> ⁉️

can you please elaborate your concern?

Reply:
In the description, you should use only text content (as a string). Do not use language keywords and syntax.